api/server/router/swarm/cluster_routes.go

```diff
@@ -107,7 +107,12 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 		return err
 	}
 
-	id, err := sr.backend.CreateService(service)
+	encodedAuth := ""
+	if auth, ok := r.Header["X-Registry-Auth"]; ok {
```

Comment:
They're received here in the capped form, even if I pass them without caps
earlier.
On Thu, Jun 16, 2016 at 19:15 Sebastiaan van Stijn <notifications@github.com>
wrote:

> In api/server/router/swarm/cluster_routes.go
> <https://github.com/docker/docker/pull/23584#discussion_r67453057>:
>
> > @@ -107,7 +107,12 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
> >  		return err
> >  	}
> >
> > -	id, err := sr.backend.CreateService(service)
> > +	encodedAuth := ""
> > +	if auth, ok := r.Header["X-Registry-Auth"]; ok {
>
> Headers should be case-insensitive; should we take that into account here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/docker/pull/23584/files/2d735640d9b9da6ac26ba7bd708214f148f0ef95#r67453057>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABySmRL_3nq3rOHY-qYuxRr9XwxMvERyks5qMgNUgaJpZM4I2rmL>
> .
>